view/frontend/requirejs-config.js

```diff
@@ -7,10 +7,10 @@ let config = {
         },
     },
     paths: {
-        bold_braintree_fastlane_client: 'https://js.braintreegateway.com/web/3.101.0-fastlane-beta.7.2/js/client.min',
-        bold_braintree_fastlane: 'https://js.braintreegateway.com/web/3.101.0-fastlane-beta.7.2/js/fastlane.min',
-        bold_braintree_fastlane_data_collector: 'https://js.braintreegateway.com/web/3.101.0-fastlane-beta.7.2/js/data-collector.min',
-        bold_braintree_fastlane_hosted_fields: 'https://js.braintreegateway.com/web/3.101.0-fastlane-beta.7.2/js/hosted-fields.min',
+        bold_braintree_fastlane_client: 'https://js.braintreegateway.com/web/3.105.0/js/client.min',
+        bold_braintree_fastlane: 'https://js.braintreegateway.com/web/3.105.0/js/fastlane',
+        bold_braintree_fastlane_data_collector: 'https://js.braintreegateway.com/web/3.105.0/js/data-collector.min',
+        bold_braintree_fastlane_hosted_fields: 'https://js.braintreegateway.com/web/3.105.0/js/hosted-fields.min',
         bold_paypal_fastlane_client: 'https://js.braintreegateway.com/web/3.97.3-connect-alpha.6.1/js/client.min',
```

Comment:
We should confirm if these should be upgraded to latest versions

Reply:
We were suggested from PayPal side to upgrade to the latest versions on the last meeting.

Reply:
Sorry, what I was meaning is the two paypal scripts at the end of this array are still pointing at the alpha versions instead of the latest versions paypal got us to upgrade to for braintree (3.105.0).

Reply:
PayPal sandbox is broken for me at the moment, so I cannot add PPCP to my m1 store to verify what versions are loaded at the moment by PayPal script. Will check and update ASAP

Reply:
Versions are the same. Here are the same components but loaded by PayPal itself
<img width="1502" alt="Screenshot 2024-08-01 at 17 09 45" src="https://github.com/user-attachments/assets/1758f2a3-567e-43bb-b36e-fb5faf0a2020">